### PR TITLE
Add note for allow attribute

### LINKED ISSUE #244: allow attribute and fetch/process
In the list under "The appropriate times to fetch and process this type of link is: "

Shouldn't we also mention the allow attribute of an iframe being changed? I mean, if allow="monetization" is removed the monetization stream should end, right?

### PR DESCRIPTION
**Pull request by huijing**
Closes #244 

This PR adds a note that explains the adding or removing the monetization attribute has no effect on an already-loaded document.

**🎩 Top-hatting instructions (i.e. human testing that things are not broken)**
1. Pull down this branch
2. Navigate to http://localhost:3000/specification.html#permissions-policy and check that the note is rendering correctly (or you could probably use the deploy preview since we turned that on ¯\\\_(ツ)_/¯)

### CODE REVIEW

File: static/specification/index.html [hunk 1]
```diff
@@ -1,1853 +1,1856 @@
...
-    </p>
-    <dl>
-      
-      <dt>Latest published version:</dt><dd>
-              <a href="https://www.w3.org/web-monetization/">https://www.w3.org/web-monetization/</a>
-            </dd>
-      <dt>Latest editor's draft:</dt><dd><a href="https://webmonetization.org/specification">https://webmonetization.org/specification</a></dd>
-      
-      
+  <meta charset="utf-8">
```
Comment (huijing) [inline static/specification/index.html:2]: unfortunately, this file got prettified 🙇🏻‍♀️

**Comment by sublimator** [comment]
Prettier them all during a lull?

**Comment by AlexLakatos** [comment]
I think we need to add the same note in the docs as well.

**Review by AlexLakatos — APPROVED** [review]
LGTM 📓

**Pull request merged**